apps/src/netsim/commands.js

```diff
@@ -0,0 +1,178 @@
+/* jshint
```

Comment:
May be worth pulling this file up into e.g. `src/` `src/utils` or `src/commands` now to communicate intent to not add netsim-specific logic to this level